text/src/autogluon/text/automm/utils.py

```diff
@@ -775,6 +778,29 @@ def create_model(
         raise ValueError(f"No available models for {names}")
 
 
+def apply_model_adaptation(model: nn.Module, config: DictConfig) -> nn.Module:
+    """
+    Apply an adaptation to the model for efficient fine-tuning.
+    Parameters
+    ----------
+    model
+        A PyTorch model.
+    config:
+        A DictConfig object. The optimization config should be accessible by "config.optimization".
+    """
+    if "lora" in config.optimization.efficient_finetune:
```

Comment:
Change `config.optimization.efficient_finetune` to `OmegaConf.select(config, "optimization.efficient_finetune")`